README.md

```diff
@@ -1,79 +1,58 @@
-
-The driver won't work with the current version of Docker Machine. To make it work, apply the provided patch :
-
-```bash
-git am --signoff < ssh_fix.patch
-```
+You need a Grid5000 account to use this driver. See [this page](https://www.grid5000.fr/mediawiki/index.php/Grid5000:Get_an_account) to create an account.
 
 ## Installation
-*The procedure was only tested on Ubuntu 16.04.*
+*This procedure was only tested on Ubuntu 16.04.*
 
-### Local host
 To use the Go tools, you need to set your [GOPATH](https://golang.org/doc/code.html#GOPATH) variable environment.
 
-To compile the binary, run:
+To get the code and compile the binary, run:
 
 ```bash
-mkdir $GOPATH/src
-mv the/sources/dir/docker-machine-driver-g5k $GOPATH/src
-go get -d -u github.com/docker/machine
-cd $GOPATH/src/docker-machine-driver-g5k
-go install
+go get -u github.com/Spirals-Team/docker-machine-driver-g5k
 ```
 
 Then, either put the driver in a directory filled in your PATH environment variable, or run:
 
 ```bash
-export PATH=/my/driver/dir/:$PATH
+export PATH=$PATH:$GOPATH/bin
 ```
 
-Be sure that your SSH public keys file is `$HOME/.ssh/id_rsa.pub`
+## How to use
 
-### Grid5000
-You should have an `authorized_keys` file in your SSH directory. Put it on your public directory, on each site you wish to create a machine on.
+### VPN
+You need to be connected to the Grid5000 VPN to create and access your Docker node.
+Dont forget to configure your DNS or use OpenVPN DNS auto-configuration.
+Please follow the instructions on the [Grid5000 Wiki](https://www.grid5000.fr/mediawiki/index.php/VPN).
 
-## How to use
 ### Options
-The driver will need a few options to create a machine. Here is a list of options:
-
-|       Option      |  Description     |  Default value   | Required  |
-|-------------------|------------------|------------------|-----------|
-| `--g5k-username`  | User's account   |                  | Yes       |
-| `--g5k-passwd`    | User's passwd    |                  | Yes       |
-| `--g5k-site`      | Site's location of the machine |    | Yes       |
-| `--g5k-walltime`  | Timelife of the machine | "1:00:00" | No        |
+The driver need a few options to create a machine. Here is a list of options:
+
+|          Option          |              Description              |     Default value     |  Required  |
+|--------------------------|---------------------------------------|-----------------------|------------|
+| `--g5k-username`         | Your Grid5000 account username        |                       | Yes        |
+| `--g5k-password`         | Your Grid5000 account password        |                       | Yes        |
+| `--g5k-site`             | Site to reserve the resources on      |                       | Yes        |
+| `--g5k-walltime`         | Timelife of the machine               | "1:00:00"             | No         |
+| `--g5k-ssh-private-key`  | Path of your ssh private key          | "~/.ssh/id_rsa"       | No         |
```

Comment:
`--g5k-ssh-private-key` is optional but I got errors while using the default value (file cannot be found). It works well if I set the parameter (`--g5k-ssh-private-key ~/.ssh/id_rsa`)